tests/baselines/reference/importAssertion2(module=commonjs).errors.txt

```diff
@@ -0,0 +1,34 @@
+tests/cases/conformance/esnext/importAssertion/1.ts(1,22): error TS2803: Import assertions are only supported when the '--module' flag is set to 'esnext'.
+tests/cases/conformance/esnext/importAssertion/1.ts(2,28): error TS2803: Import assertions are only supported when the '--module' flag is set to 'esnext'.
+tests/cases/conformance/esnext/importAssertion/1.ts(3,21): error TS2803: Import assertions are only supported when the '--module' flag is set to 'esnext'.
+tests/cases/conformance/esnext/importAssertion/1.ts(4,27): error TS2803: Import assertions are only supported when the '--module' flag is set to 'esnext'.
+tests/cases/conformance/esnext/importAssertion/2.ts(1,28): error TS2803: Import assertions are only supported when the '--module' flag is set to 'esnext'.
+tests/cases/conformance/esnext/importAssertion/2.ts(2,38): error TS2803: Import assertions are only supported when the '--module' flag is set to 'esnext'.
+
+
+==== tests/cases/conformance/esnext/importAssertion/0.ts (0 errors) ====
+    export const a = 1;
+    export const b = 2;
+    
+==== tests/cases/conformance/esnext/importAssertion/1.ts (4 errors) ====
+    export {} from './0' assert { type: "json" }
+                         ~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS2803: Import assertions are only supported when the '--module' flag is set to 'esnext'.

```

Comment:
Perhaps its just be, but the term "Import assertion" seems strange when applied to an `export` declaration. While I understand its the name of the proposal in question, I wonder if there's a less-specific term we could use here.

/CC @DanielRosenwasser

Reply:
how about `import or export assertion`